web/controllers/care_team/room_controller.ex

```diff
@@ -4,7 +4,7 @@ defmodule Healthlocker.CareTeam.RoomController do
 
   def show(conn, %{"id" => id}) do
     room = Repo.get! assoc(conn.assigns.current_user, :rooms), id
-    service_user = service_user_for(conn.assigns.current_user)
+    service_user = Healthlocker.CareTeam.ContactController.service_user_for(conn.assigns.current_user)
```

Comment:
If a function needs to be shared across controllers we usually prefer to put them in small modules, this keeps our controllers small and focused and makes the modules easily shareable and testable. 

If you look in https://github.com/healthlocker/healthlocker/blob/read-receipts/lib/healthlocker/slam/care_team.ex#L9 I've already extracted the `care_team_for` function (however, I should have also removed it from here doh!)

I'd suggest we do something similar for the `service_user_for` function.